src/sql/workbench/parts/notebook/electron-browser/outputs/notebookMarkdown.ts

```diff
@@ -59,7 +60,7 @@ export class NotebookMarkdownRenderer {
 		let signalInnerHTML: () => void;
 		const withInnerHTML = new Promise(c => signalInnerHTML = c);
 
-		let notebookFolder = path.dirname(this._notebookURI.path) + '/';
+		let notebookFolder = path.dirname(this._notebookURI.fsPath) + '/';
```

Comment:
Can we do path.join here? As I recall there's a property in path you can avoid hardcoding '/'

Reply:
Yes, we should be using path.sep for this. 

---
In reply to: [323447857](https://github.com/microsoft/azuredatastudio/pull/7158#discussion_r323447857) [](ancestors = 323447857)